lib/createlocaltrack.js

```diff
@@ -92,7 +95,14 @@ function createLocalVideoTrack(options) {
  * also specify any of the <a href="https://developer.mozilla.org/en-US/docs/Web/API/MediaTrackConstraints" target="_blank">MediaTrackConstraints</a>
  * properties.
  * @typedef {MediaTrackConstraints} CreateLocalTrackOptions
- * @property {LogLevel|LogLevels} logLevel
```

Comment:
I think we should support the loglevel (but mark it as deprecated)

Reply:
I don't think we should mark this as deprecated because we don't plan on removing this functionality, including the ability to specify different levels for each module. Let's hear what @manjeshbhargav has to say.

Reply:
@charliesantos The goal is to replace the existing log functionality with the new logger in the next major version. So, we can mark the `logLevel` parameter as deprecated and ask them to use the new logger instead.

Reply:
@charliesantos You should also print a console warning if `logLevel` is specified in ConnectOptions.